dashboard/templates/base.html

```diff
@@ -8,9 +8,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{% block title %}{% endblock %}</title>
-    <link rel="stylesheet" type="text/css" href="{% static '@fortawesome/fontawesome-free/css/all.css' %}">
-    <link rel="stylesheet" type="text/css" href="{% static '@fortawesome/fontawesome-free/css/fontawesome.css' %}">
-    <link rel="stylesheet" type="text/css" href="{% static '@fortawesome/fontawesome-free/css/fontawesome.min.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'fontawesomefree/css/all.min.css' %}">
+    <script src="{% static 'fontawesomefree/js/all.min.js' %}"></script>
```

Comment:
`fontawesome` offers a Django/solution for using icons in templates called `fontawesomefree`: https://pypi.org/project/fontawesomefree/ This isn't strictly a replacement for `django-npm`, but after we removed `jquery`, `fontawesome` seemed to be the only thing we really needed `django-npm` for.

To explain a bit more, `fontawesome` CSS classes are being used for Resources Accessed and for screenreader-only content, and thus we needed a way to load the styling. Using a strictly React approach proved somewhat challenging because of 1) we currently use string classes for icons on the Resources Accessed D3 chart, where we can't really use JSX and 2) figuring out how to load the CSS file through `webpack` was a headache. If D3 is ever replaced, we could look into using this approach (https://fontawesome.com/docs/web/use-with/react/), or removing `fontawesome` all together in favor of Material UI icons. The latter though would require configuration changes, since we currently pass CSS class strings in `env.json` for each resource type.

@jonespm, let me know what you think of this solution for removing `django-npm`. Here's the commit:
https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1363/commits/eb3492de43acc1752d5e587c74523855f90ddc28

Reply:
I think originally pre-react we needed this for more. But that commit looks good to remove it. I'd rather have this supported package than the older unsupported one. Be great if we have time to eventually do something with D3.